### DISCUSSION TITLE
Update HelmRelease within the git repository

### DISCUSSION THREAD

**Discussion by OferPRTZ** [fluxcd/flux2#322] [Q&A] [answered]
Hi Guys,

Newbie here with the gitops toolkit.
Ive been searching the Gitops Toolkit guides / docs for sometime now how i can use the toolkit to update my HelmRelease inside my git repo.
Steps ive made:
`gotk bootstrap gitlab --owner=devops --repository=fluxv2 --path=test-cluster --hostname=my.gitlab.url --branch master`
`gotk create source helm myrepo --url=my.chart-museum.url --interval=1m`

My git repo
├── apps
│   └── test-helmrelease-app1.yaml
│   └── test-helmrelease-app2.yaml
│ 
├── gitops-system
│   ├── toolkit-components.yaml
│   └── toolkit-kustomization.yaml
│   └── toolkit-source.yaml

Then ive created HelmRelease manually to deploy my app with the HelmRepository and it works fine.
my question is how can i get the toolkit to listen on my git repository where the helmrelease yaml reside and act upon changes?

Thanks,
Ofer

**Comment by stefanprodan** [comment, marked as answer]
You have to place the app definitions inside the `test-cluster` dir.

**Reply by OferPRTZ** [reply]
Thanks Stefan! 
That really helped me

Will be great if you guys have time to add some directory structure or a beginner's guide as you did in fluxv1

cheers
